src/DetailsView/components/target-page-changed-view.tsx

```diff
@@ -11,11 +13,16 @@ export interface TargetPageChangedViewProps {
     visualizationType: VisualizationType;
     displayableData: DisplayableVisualizationTypeData;
     toggleClickHandler: (event) => void;
+    featureFlagStoreData: FeatureFlagStoreData;
 }
 
 export const TargetPageChangedView = NamedFC<TargetPageChangedViewProps>('TargetPageChangedView', props => {
     const { title = '', toggleLabel = '', subtitle } = props.displayableData;
 
+    const oldText = 'The target page was changed. Use the toggle to enable the visualization in the current target page.';
```

Comment:
Could we name this something more informative than 'oldText'? :)

Reply:
Good point! I wasn't sure what to rename it to: any ideas?

Maybe toggleBasedText and startOverBasedText?